### DISCUSSION TITLE
postgres Connection terminated unexpectedly

### DISCUSSION THREAD

**Discussion by KingDracula** [sequelize/sequelize#15524] [Help & Questions]
when i connection postgres ,sometime was error Connection terminated unexpectedly 
retry it is work ,
i try to update pool config ,but not work 
   "define": {
            "schema": "public"
        },
        "dialectOptions": {
            "useUTC": true,
            "timezone": "UTC",
            "connectionTimeout": 60000,
            "requestTimeout": 60000
        },
        "pool": {
            "max": 100,
            "min": 5,
            "idle": 300000,
            "acquire": 300000
        },
        "idleTimeoutMillis": 0,
        "connectionTimeoutMillis": 0,

Please help me solve this problem

**Comment by tejasautkar** [comment]
Hi, any luck on this? I too am getting same error.
